### PR TITLE
server can now send sms + time synchronization

### PR DESCRIPTION
**Pull request by vallettea**

### CODE REVIEW

File: app/client/browserify-bundle.js [hunk 1]
```diff
@@ -1,4 +1,12 @@
...
+(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({"/6element/app/PRIVATE.json":[function(require,module,exports){
+module.exports={
+	"twilio_ssi": "AC152071c164a7b0324437b7d6422bd88b",
+	"twilio_pwd": "2591d9aadcfe1de083681142887d2cd5",
+	"twilio_number": "+33644640016",
```
Comment (DavidBruant) [inline app/client/browserify-bundle.js]: ahem...
https://help.github.com/articles/remove-sensitive-data/

Comment (iOiurson) [inline app/client/browserify-bundle.js]: the ```browserify-bundle.js``` (and equivalents) should be in ```.gitignore```, is that what you mean ?

Comment (DavidBruant) [inline app/client/browserify-bundle.js]: Yes.
And the article I linked to explains how to clean the git history so no one has access to the twilio ssi and passwords even if they look at the commit history.

Comment (vallettea) [inline app/client/browserify-bundle.js]: on it

**Pull request merged**